decidim-conferences/config/locales/en.yml

```diff
@@ -403,6 +403,8 @@ en:
         personal_website: Personal website
         show:
           more_info: more info
+          speaking_at: Speaking at
+        speaking_at: Speaking at
```

Comment:
Hello @takahashim 
why are we adding two entries for 'speaking_at' tag in locales file ? Seems that only show.speaking_at enty would be necessary?
Thanks in advance!

Reply:
Thanks for your comment!
At first I add `conference_speaker.speaking_at` into `en.yml` and it seems OK (and took the screenshot above). But I18N test said `conference_speaker.show.speaking_at` is missing https://github.com/decidim/decidim/runs/1401753550
I'm not sure why this duplication are needed, but another keyword `more_info` in `app/cells/decidim/conferences/conference_speaker/show.erb` is also written in `conference_speaker.more_info` and `conference_speaker.show.more_info`, so I guess I should do like that.

Reply:
Hi again @takahashim ! 
in first place, thanks for you explanation. :smile: 
Usually, translations path follow same path as template where they are used. For example, as template path is "decidim/conferences/conference_speaker/show.erb", the "speaking_at" tag would be placed in 
```
decidim:
   conferences:
      conference_speaker:
         show:
            speaking_at: 
```
I know this is picky, but you can try removing second one in "decidim.conferences.conference_speaker.speaking_at" and try if it works well? :pray: 
Thanks a lot! :smile:

Reply:
OK, I'll try it later 🙆‍♀️

Reply:
@Leusev I've tried to do it.  However, simply deleting the entry `decidim.conferences.conference_speaker.speaking_at` caused an `I18n::MissingTranslationData` error.

After further investigation, it could be successfully displayed by making the following modifications. Should I apply this fix into this PR?

```diff
 $ git diff
diff --git a/decidim-conferences/app/cells/decidim/conferences/conference_speaker/show.erb b/decidim-conferences/app/cells/decidim/conferences/conference_speaker/show.erb
index c7902382d..1f36da0e7 100644
--- a/decidim-conferences/app/cells/decidim/conferences/conference_speaker/show.erb
+++ b/decidim-conferences/app/cells/decidim/conferences/conference_speaker/show.erb
@@ -70,7 +70,7 @@
       <hr class="reset mt-none mb-s">
       <div class="row">
         <div class="column medium-12">
-          <div class="bio-acts"><%= t(".speaking_at") %></div>
+          <div class="bio-acts"><%= t("conferences.conference_speaker.show.speaking_at", scope: "decidim") %></div>
 
           <ul class="list-reset">
             <% meetings.each do |meeting| %>
diff --git a/decidim-conferences/config/locales/en.yml b/decidim-conferences/config/locales/en.yml
index b1dfe5474..c0e6a5626 100644
--- a/decidim-conferences/config/locales/en.yml
+++ b/decidim-conferences/config/locales/en.yml
@@ -404,7 +404,6 @@ en:
         show:
           more_info: more info
           speaking_at: Speaking at
-        speaking_at: Speaking at
       conference_speaker_cell:
         personal_url:
           personal_website: Personal website
```

Reply:
Hi @takahashim 
yeah, it's a weird thing what was happening to you.. But for me side, I look well your proposed fix :smile: 
Yes please, apply it and all will be ok for this PR
Thanks a lot for your time! :+1:

Reply:
Thanks for your commment, I've add a commit.